projects/kubernetes/image-cache/Dockerfile

```diff
@@ -1,5 +1,22 @@
-FROM linuxkit/docker-ce:9b937df179bdbebbc70243779978057df0b54190
-ADD . /images
+FROM linuxkit/docker-ce:9b937df179bdbebbc70243779978057df0b54190 AS docker
```

Comment:
this may merge conflict with the PR to not use our docker image; also the `dind` image has the docker binary in `/usr/local/bin` so above will need fixing...